library/src/main/res/values/colors.xml

```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+    <color name="emoji_background">#ECEFF1</color>
+    <color name="emoji_icons">#61000000</color>
+    <color name="emoji_divider">#15000000</color>
```

Comment:
I can do that, but it does not look very good together with the other default colors IMHO. This value is calculated by the same method I described above, for the recommended value (Dark text - Dividers) of 12% opacity.